functions/fish_prompt.fish

```diff
@@ -846,6 +847,47 @@ function __bobthefish_prompt_rubies -S -d 'Display current Ruby information'
     echo -ns $ruby_glyph $ruby_version ' '
 end
 
+function __bobthefish_get_cargo_version -S -d 'Display the cargo version in a directory'
+    pushd $argv[1]
+    command rustc --version | cut -d ' ' -f 2
```

Comment:
Does rustc take a working directory to save us from a pushd/popd? Otherwise, we should probably add `command`.

Reply:
Unfortunately no. I'll add `command` but I'm also exploring if there's a better way to get this information.

Reply:
So I decided to rely on `fish -c " ... "` to create a subshell only if we need to (which should be once for the global anyway).